en/mulesoft/modules/ROOT/pages/ms_composer_sage_intacct_reference.adoc

```diff
@@ -0,0 +1,225 @@
+
+Create Customer::
+
+Create a new Customer record. Provide a value for the following:
+
+* *Customer ID*
+* *Primary Contact*
+* *Print As*
+
+Update Customer::
+
+Update a Customer record. Provide a value for *ID*.
+
+Get Customer::
+
+Returns a list of Customer records. To filter the list, provide a value for *Customer ID* or *Customer Name*. To return the entire list, leave the fields blank.
+
+Delete Customer::
+
+Deletes a Customer record. Provide a value for *Customer ID*.
+
+Create Order::
+
+Creates a new Order record and returns a value for the *Key* field. Provide a value for the following:
+
+* *Order Type*
+* *Date Created*
+* *Transaction Currency*
+* *Base Currency*
+* *Exchange Rate Type*
+* In the *Entries* > *Sotransitem* section, provide a value in the following:
+  ** *Sotransitem List*
+  ** *Item ID*
+  ** *Quantity*
+  ** *Location ID* +
+Note that the *Location ID* field is determined by the numbers preceding the location description. For example, if the location ID is `100-United States` then enter `100` in the *Location ID* field.
+
+  ** *Price*
+  ** *Unit*
+  ** If your order contains inventory type items or a mix of inventory and non-inventory items, provide a value for *Warehouse ID*.
+
+Update Order::
+
+Updates an Order record. +
+Note that when updating or reordering line items for an existing order, the line order can change. For example, if you drag a line item from the number two position to the number three position in the Sage Intacct UI, the line order changes. To avoid synchronizing the wrong Sage Intacct line order item, you must account for possible order line item changes. Prior to executing the *Update Order* action, identify the current *Line Number* and store it in a custom field within Sage Intacct. To delete a line item from an existing order, use the *Delete Order Line Item* action. +
+Provide a value for the following:
+
+* *Document ID* +
+Note that the *Document ID* value is the *Key* value returned by the *Create Order* action.
+
+*  In the *Entries* > *Updatesotransitem* section, provide a value for the following:
+** *Sotransitem List*
+** *Item ID*
+** *Quantity*
+** *Location ID* +
+Note that the *Location ID* field is determined by the numbers preceding the location description. For example, if the location ID is `100-United States` then enter `100` in the *Location ID* field.
+
+** *Price*
+** *Unit*
+** If your order contains inventory type items or a mix of inventory and non-inventory items, provide a value for *Warehouse ID*. +
+
+Delete Order Line Item::
+
+Deletes an order line item. Provide a value for *Line Item ID (Record Number)*.
+
+Get Order::
+
+Returns a list of Order records. To filter the list, provide a value for *Order ID*, *Customer ID* or *Document ID*. To return the entire list, leave the fields blank. Note that the *Document ID* value is the *Key* value returned by the *Create Order* action.
```

Comment:
For consistency, add a blank line before the text that starts with "Note that..."